tests/api/test_v1_push.py

```diff
@@ -9,6 +10,12 @@
 @pytest.mark.django_db
 class TestV1PushProxy:
 
+    @pytest.fixture(autouse=True)
+    def mock_disambiguate(self):
```

Comment:
It's automatically used for all tests in that class due to `autouse=True`. It prevents celery from attempting to contact rabbit.